src/NzbDrone.Core/HealthCheck/HealthCheckRestoredEvent.cs

```diff
@@ -0,0 +1,16 @@
+using NzbDrone.Common.Messaging;
+
+namespace NzbDrone.Core.HealthCheck
+{
+    public class HealthCheckRestoredEvent : IEvent
+    {
+        public HealthCheck PreviousCheck { get; private set; }
+        public bool IsInStartupGraceperiod { get; private set; }
```

Comment:
`IsInStartupGracePeriod` eventually we should fix the health issue event as well.

Reply:
Done, should I fix the other one whilst we're here or leave it?

Reply:
You can leave it, I've already fixed it locally.